sql/testdata/builtin_function

```diff
@@ -573,4 +575,4 @@ SELECT translate('12345', 'abc', 'ax')
 query T
 SELECT translate('a‰ÒÁ', 'aÒ', '∏p')
 ----
-∏‰pÁ
\ No newline at end of file
+∏‰pÁ
```

Comment:
Don't worry about it. Many editors add trailing newlines automatically, and it's probably better to have them than not (if nothing else, the `diff` output is ugly if there is no newline)